src/poli/objective_repository/gfp_cbas/cbas_wrapper.py

```diff
@@ -76,7 +76,7 @@ def decode(self, z, grad=False):
         else:
             return self._decoder(z)
 
-    def transform_sequences_to_atoms(self, x: [torch.Tensor]) -> torch.Tensor:
+    def transform_sequences_to_atoms(self, x: torch.Tensor) -> torch.Tensor:
         """
         Turns a sequence into an approximately Dirac distribution around the sequence.
         :param x:
```

Comment:
We're currently following [`numpy` standards for documentation](https://numpydoc.readthedocs.io/en/latest/format.html). But don't worry about this! I'll change them later on.

Reply:
Ah, I see. I've changed it to comply with my VSC recommendations. Can also change my syntax checker.